proto/zetachain/zetacore/crosschain/last_block_height.proto

```diff
@@ -7,6 +7,6 @@ message LastBlockHeight {
   string creator = 1;
   string index = 2;
   string chain = 3;
-  uint64 lastSendHeight = 4;
-  uint64 lastReceiveHeight = 5;
+  uint64 lastInboundHeight = 4;
```

Comment:
Is this stateful`LastBlockHeight` going to be a piece of deprecated data? I don't see it's used anywhere in zetaclient.

Reply:
https://github.com/zeta-chain/node/issues/2216